src/main/java/tc/oc/pgm/spawns/states/Observing.java

```diff
@@ -109,4 +111,13 @@ public void onEvent(EntityDamageEvent event) {
     super.onEvent(event);
     event.setCancelled(true);
   }
+
+  @Override
+  public void onEvent(PlayerInteractEvent event) {
+    super.onEvent(event);
+    if (PickerMatchModule.validatePickerItem(event.getPlayer().getItemInHand())) {
```

Comment:
Seems weird for `PickerMatchModule` to expose this implementation detail. How about add a `ObserverInteractEvent` listener to `PickerMatchModule` and cancel the event.